AlphaWallet/Transfer/ViewModels/SendHeaderViewViewModel.swift

```diff
@@ -8,13 +8,19 @@ struct SendHeaderViewViewModel {
     var currencyAmount: String?
     var currencyAmountWithoutSymbol: Double?
     var showAlternativeAmount = false
+    var chainId = Config().chainID
 
     var issuer: String {
         return ""
     }
 
     var blockChainName: String {
-        return R.string.localizable.blockchainEthereum()
+        if chainId == 100 {
```

Comment:
I think it's better to use a switch right away. Much easier to discover this piece of code rather than remember to search for 100 later.